activities/communication/blogging.md

```diff
@@ -0,0 +1,98 @@
+
+Blogposts are different from [resources and guides published on About](../documentation/index.md), like a white paper we keep updated. By contrast, a blogpost is only true at that moment in time. We shouldn’t ever update a blogpost with more current information.
+
+Instead, we can add a note with a link to follow up blogposts or new information published on the same topic. Adding links from older blogposts to newer blogposts also lets new blogposts benefit from the greater search engine weighting given to older, still popular pages.
+
+This approach reduces our [future content debt](https://18f.gsa.gov/2016/05/19/content-debt-what-it-is-where-to-find-it-and-how-to-prevent-it-in-the-first-place/) and avoids undermining our position of authority - for example, we'll never have to change a blogpost to say that due to a government policy shift, an expert we’ve interviewed is no longer in post (which might cast doubt on that expert’s still-relevant advice).
+
+## Before you start
+
+1. Since none of us work in isolation, check in with the communications coordinator before you start writing your blogpost. This helps us:
+  * understand your goals and audience for the blogpost
+  * establish the scope of your blogpost (to avoid duplication with other planned work)
+  * schedule publication of your blogpost so it fits best with other planned communications
+
+2. [Read our style guide](https://content-guide.18f.gov/) - we use 18F's. If you have any questions, ask!
+3. Make sure your information is correct on [blog/_data/staff.csv](https://github.com/publiccodenet/blog/blob/develop/_data/staff.csv).
+4. Read the [guide on writing community call blogposts](community-call-blogposts.md) if that's what you're doing.
+
+## Write your blogpost
+
+Write your draft blogpost using a collaborative document tool. This lets you share it for pair writing, factchecking, feedback or proofreading.
+
+Don't hesitate to involve other people in your writing process - we want you to succeed, and we know how helpful an external perspective can be once something starts to feel like a slog.
+
+After writing, please share with the communications coordinator for polishing.
+
+## How to create a blogpost on the Foundation’s blog
+
+### Step 1: create your branch and file
+
+Create a branch for your new blogpost within in the [blog repo](https://github.com/publiccodenet/blog).
+
+Create a new file for your blogpost within [`_posts/` on the blog repository](https://github.com/publiccodenet/blog/tree/develop/_posts)
+named `YYYY-MM-DD-title.md`.
+
+### Step 2: create your blog post
+
+At the top of the file you should have this frontmatter:
+
+```markdown
+---
+title:  "Title”
+date: YYYY-MM-DD
+author: [put your name]
+type: blogpost
+category: [our current categories are: news, community call and codebase stewardship]
+excerpt: [unformatted text you want to appear on the index]
+---
+```
+
+All blogposts should include at least one relevant image, to make them more engaging.
```

Comment:
All blogposts should include at least one relevant image or video to make them more engaging.

Reply:
Done!